app/migrations/versions/456eba67de05_.py

```diff
@@ -18,9 +18,17 @@
 depends_on = None
 
 
+def index_exists(name):
```

Comment:
This change needs to be present for a deploy to work. It's already deployed in Pacific, but is still waiting for a review here: https://github.com/teamsempo/SempoBlockchain/pull/584